### ISSUE TITLE
关于TimesNet的周期计算

### ISSUE THREAD

**Issue by chenzehao82** [thuml/Time-Series-Library#434]
def FFT_for_Period(x, k=2):
    # [B, T, C]
    xf = torch.fft.rfft(x, dim=1)
    # find period by amplitudes
    frequency_list = abs(xf).mean(0).mean(-1)
    frequency_list[0] = 0
    _, top_list = torch.topk(frequency_list, k)
    top_list = top_list.detach().cpu().numpy()
    period = x.shape[1] // top_list
    return period, abs(xf).mean(-1)[:, top_list]
你这里top_list是坐标，怎么能通过坐标计算的period

**Comment by wuhaixu2016** [comment]
您好，top_list的id对应频率，总长度/频率=周期，请见https://pytorch.org/docs/stable/generated/torch.fft.rfft.html
